### PR TITLE
Setting Up DB Schema

### PR DESCRIPTION
**Pull request by calvarezberrios**
Edititing schema for profiles table, added stores, stores_reviews, and products tables. Also added seeds for these tables.

### CODE REVIEW

**Comment by calvarezberrios** [comment]
[Create Schema and Seed Data as per updated db schema model](https://trello.com/c/Mj7ytNzX/28-create-schema-and-seed-data-as-per-updated-db-schema-model)

File: data/seeds/04_photo.js [hunk 1]
```diff
@@ -1,156 +0,0 @@
-exports.seed = function (knex) {
```
Comment (DiplomaZ) [inline data/seeds/04_photo.js:1]: Do you think we could include these seeds for the existing code, just for the time being? We can remove them if they wind up causing conflicts as our code evolves.

**Review by calvarezberrios — COMMENTED** [review]
Will leave this branch alone and update on the existing migrations and seeds

**Comment by calvarezberrios** [comment]
Not deleting branch in case we decide to use these changes

**Pull request closed without merging**